<?xml version="1.0" encoding="UTF-8"?>
<testsuites>
    <testsuite name="force.apex" timestamp="2019-12-12T21:47:08-06:00" hostname="https://customization-business-8920-dev-ed.cs94.my.salesforce.com" tests="1" failures="1"  errors="0"  time="0.04">
        <properties>
            <property name="outcome" value="Failed"/>
            <property name="testsRan" value="1"/>
            <property name="passing" value="0"/>
            <property name="failing" value="1"/>
            <property name="skipped" value="0"/>
            <property name="passRate" value="0%"/>
            <property name="failRate" value="100%"/>
            <property name="testStartTime" value="Dec 12, 2019 9:47 PM"/>
            <property name="testExecutionTime" value="0.04 s"/>
            <property name="testTotalTime" value="0.04 s"/>
            <property name="commandTime" value="5.81 s"/>
            <property name="hostname" value="https://customization-business-8920-dev-ed.cs94.my.salesforce.com"/>
            <property name="orgId" value="00D0R0000001hlnUAA"/>
            <property name="username" value="test-tkg6ixvrkmnr@example.com"/>
            <property name="testRunId" value="7070R00000mLwi2"/>
            <property name="userId" value="0050R000003YmuIQAS"/>
        </properties>
        <testcase name="shouldDeleteForm" classname="FormControllerTest" time="0.04">
            <failure message="System.AssertException: Assertion Failed: Expected: Unable to delete form while it has Responses, Actual: Script-thrown exception"><![CDATA[Class.FormControllerTest.shouldDeleteForm: line 46, column 1]]></failure>
        </testcase>
    </testsuite>
</testsuites>
